solutions/standard/variables.tf

```diff
@@ -215,13 +215,13 @@ variable "kms_endpoint_type" {
 
 variable "kms_key_ring_name" {
   type        = string
-  default     = "sm-cos-key-ring"
+  default     = "secrets-manager-cos-key-ring"
```

Comment:
cos should not be in the name at all. This key used used for encrypting the secrets manager data. There is no COS in this DA. Suggest to rename `secrets-manager-key-ring`